### ISSUE TITLE
 Unable to set cursor to arrow: cursor not found

### ISSUE THREAD

**Issue by dmichalskii** [wez/wezterm#2703]
### What Operating System(s) are you seeing this problem on?

Linux Wayland

### Which Wayland compositor or X11 Window manager(s) are you using?

Sway, Gnome 42

### WezTerm version

20220905-102802-7d4b8249

### Did you try the latest nightly build to see if the issue is better (or worse!) than your current version?

Yes, and I updated the version box above to show the version of the nightly that I tried

### Describe the bug

The problem is:
`
17:25:44.330  ERROR  window::os::wayland::pointer > Unable to set cursor to arrow: cursor not found
`
I need to reinstall system and after reinstall I see this error... Never see problem with wezterm before. Don't know what can cause problem?

### Configuration

no config

### Expected Behavior

Working without this error, because now when I move mouse (some time to time) I see peeks in 

### Anything else?

I am using wezterm long time without issues.

**Comment by wez** [comment]
Try starting wezterm like this:

```
WEZTERM_LOG=window::os::x11::cursor=trace,window::os::wayland::pointer=trace,info wezterm
```

and share the logs.

See also: https://wezfurlong.org/wezterm/faq.html#i-use-x11-or-wayland-and-my-mouse-cursor-theme-doesnt-seem-to-work

**Comment by dmichalskii** [comment]
`
WEZTERM_LOG=window::os::x11::cursor=trace,window::os::wayland::pointer=trace,info flatpak run org.wezfurlong.wezterm
`
same log as before with error.

**Comment by wez** [comment]
I think this is fixed by 9569028d98e6d07256901d3c5f971aaae3b77c6f but you probably won't be able to test until the next release is cut.
This issue is a duplicate of https://github.com/wez/wezterm/issues/2687
